proto/kujira/oracle/genesis.proto

```diff
@@ -10,12 +10,9 @@ option go_package = "github.com/Team-Kujira/core/x/oracle/types";
 // GenesisState defines the oracle module's genesis state.
 message GenesisState {
   Params                     params             = 1 [(gogoproto.nullable) = false];
-  repeated FeederDelegation  feeder_delegations = 2 [(gogoproto.nullable) = false];
-  repeated ExchangeRateTuple exchange_rates     = 3
+  repeated ExchangeRateTuple exchange_rates     = 2
```

Comment:
Maybe we should retain the same indexes for the same properties? Make an old genesis export compatible with the new module

Reply:
Genesis encoding/decoding's done in JSON and proto index won't be affected I believe.